conversations/ts/components/ConversationListItem.tsx

```diff
@@ -57,14 +54,9 @@ export class ConversationListItem extends React.Component<Props> {
       );
     }
 
-    const title = `${name || phoneNumber}${
-      !name && profileName ? ` ~${profileName}` : ''
-    }`;
-
     return (
       <img
         className="module-conversation-list-item__avatar"
-        alt={i18n('contactAvatarAlt', [title])}
```

Comment:
I removed this since I don't really see any use of it since at this point we know we have an image. The other lines went away because not used (TypeScript compiler complains).

Reply:
yeah makes sense to me. Although I wonder if we would want to add back in some sort of 'address' in place of phone number at some point.